timber-lint/src/main/java/timber/lint/WrongTimberUsageDetector.java

```diff
@@ -66,18 +71,18 @@
     return Arrays.asList("tag", "format", "v", "d", "i", "w", "e", "wtf");
   }
 
-  @Override public void visitMethod(JavaContext context, UCallExpression call, PsiMethod method) {
+  @Override public void visitMethodCall(JavaContext context, UCallExpression call, PsiMethod method) {
     String methodName = call.getMethodName();
     JavaEvaluator evaluator = context.getEvaluator();
 
     if ("format".equals(methodName) && evaluator.isMemberInClass(method, "java.lang.String")) {
       checkNestedStringFormat(context, call);
       return;
     }
-    // As of API 24, Log tags are no longer limited to 23 chars.
+    // As of API 26, Log tags are no longer limited to 23 chars.
     if ("tag".equals(methodName)
         && evaluator.isMemberInClass(method, "timber.log.Timber")
-        && context.getMainProject().getMinSdk() <= 23) {
+        && context.getProject().getMinSdk() <= 25) {
```

Comment:
so i think this will still fail.  the new Lint style is to use a constraint

Reply:
so this is no longer necessary?

Reply:
I kept it just so the check wouldn't be run in the second lint run. It should work without it, but not 100% sure.